dotnet/src/IntegrationTestsV2/Connectors/AzureOpenAI/AzureOpenAIAudioToTextTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.AudioToText;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+using SemanticKernel.IntegrationTests.TestSettings;
+using Xunit;
+
+namespace SemanticKernel.IntegrationTestsV2.Connectors.AzureOpenAI;
+
+public sealed class AzureOpenAIAudioToTextTests()
+{
+    private readonly IConfigurationRoot _configuration = new ConfigurationBuilder()
+        .AddJsonFile(path: "testsettings.json", optional: true, reloadOnChange: true)
+        .AddJsonFile(path: "testsettings.development.json", optional: true, reloadOnChange: true)
+        .AddEnvironmentVariables()
+        .AddUserSecrets<AzureOpenAIAudioToTextTests>()
+        .Build();
+
+    [Fact]
+    public async Task AzureOpenAIAudioToTextTestAsync()
+    {
+        // Arrange
+        const string Filename = "test_audio.wav";
+
+        AzureOpenAIConfiguration? azureOpenAIConfiguration = this._configuration.GetSection("AzureOpenAIAudioToText").Get<AzureOpenAIConfiguration>();
+        Assert.NotNull(azureOpenAIConfiguration);
+
+        var kernel = Kernel.CreateBuilder()
+            .AddAzureOpenAIAudioToText(
+                azureOpenAIConfiguration.DeploymentName,
+                azureOpenAIConfiguration.Endpoint,
+                azureOpenAIConfiguration.ApiKey)
+            .Build();
+
+        var service = kernel.GetRequiredService<IAudioToTextService>();
+
+        await using Stream audio = File.OpenRead($"./TestData/{Filename}");
```

Comment:
Curious why are we using `await` here if the method `OpenRead` is not async?

```suggestion
        await using Stream audio = File.OpenRead($"./TestData/{Filename}");
```

Reply:
This test is a copy of the equivalent one for the `OpenAIAudioToTextService` service. The reason for having it, I suppose, is that the method returns a stream that supports async disposal, and Visual Studio detected this and showed a warning/suggestion to use the async disposal.